spk/minio/src/wizard/install_uifile

```diff
@@ -0,0 +1,67 @@
+[
+	{
+		"step_title": "minio configuration",
+		"items": [
+		  {
+			"type": "textfield",
+			"desc": "Data directory location",
+			"subitems": [
+			  {
+				"key": "wizard_data_directory",
+				"desc": "Data directory location",
+				"defaultValue": "/volume1/minio",
+				"validator": {
+				  "allowBlank": false,
+				  "regex": {
+					"expr": "/^\\\/volume[0-9]{1,2}\\\/[^<>: */?\"]*/",
+					"errorText": "Path should begin with /volume?/ where ? is volume number (1-99)"
+				  }
+				}
+			  },
+			  {
+				"key": "wizard_group",
+				"desc": "DSM group",
+				"defaultValue": "sc-minio",
+				"validator": {
+				  "allowBlank": false,
+				  "regex": {
+					"expr": "/^[^<>:*/?\"]*$/",
+					"errorText": "Not allowed character in group name"
+				  }
+				}
+			  },
+			  {
+				"key": "wizard_access_key",
+				"desc": "minio access key",
+				"defaultValue": "minio-access-822791549661344",
+				"validator": {
+				  "allowBlank": false,
+				  "regex": {
+					"expr": "/^[^<>:*/?\"]*$/",
+					"errorText": "Not allowed character in key"
+				  }
+				}
+			  },
+			  {
+				"key": "wizard_secret_key",
+				"desc": "minio secret key",
+				"defaultValue": "minio-secret-033309986404774",
+				"validator": {
+				  "allowBlank": false,
+				  "regex": {
+					"expr": "/^[^\"]*$/",
+					"errorText": "Not allowed character in key"
+				  }
+				}
+			  }
+			]
+		  }
+		]
+	},
+	{
+		"step_title": "Attention! DSM6 Permissions",
+		"items": [{
+			"desc": "Permissions for this package are handled by the <b>'sc-minio'</b> group. <br>Using File Station, add this group to every folder minio should be allowed to access.<br>Please read <a <a target=\"_blank\" href=\"https://github.com/SynoCommunity/spksrc/wiki/Permission-Management\">Permission Management</a> for details."
```

Comment:
a bit weird that you leave a choice to set a custom group name above but refer to the specific group `sc-minio` here

Reply:
Fixed in latest commit.